provider/terraform/tests/resource_dataproc_cluster_test.go

```diff
@@ -134,6 +134,22 @@ func TestAccDataprocCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccDataprocCluster_withAccelerators(t *testing.T) {
+	t.Parallel()
+
+	rnd := acctest.RandString(10)
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDataprocClusterDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataprocCluster_withAccelerators(rnd),
```

Comment:
Since we don't seem to have import support yet, can you add some checks to this test?